packages/morph-attr/lib/main.js

```diff
@@ -59,31 +75,19 @@ function AttrMorph(element, attrName, domHelper, namespace) {
   this.linkedParams = null;
   this.linkedResult = null;
   this.guid = "attr" + guid++;
+  this.seen = false;
   this.ownerNode = null;
   this.rendered = false;
   this._renderedInitially = false;
-
-
-  if (this.namespace) {
-    this._update = updateAttributeNS;
-    this._get = getAttributeNS;
-    this.attrName = attrName;
-  } else {
-    var { normalized, type } = normalizeProperty(this.element, attrName);
-
-    if (element.namespaceURI === svgNamespace || attrName === 'style' || type === 'attr') {
-      this._update = updateAttribute;
-      this._get = getAttribute;
-      this.attrName = normalized ;
-    } else {
-      this._update = updateProperty;
-      this._get = getProperty;
-      this.attrName = normalized ;
-    }
-  }
+  this.didInit();
```

Comment:
It allows Ember to hook in (we have a slightly customized AttrMorph class over there).